### PR TITLE
Move Community list to Contentful

### PR DESCRIPTION
**Pull request by yelyzavetakhokhlova**
### PR Checklist

* [ ] Have you followed the guidelines in our [Dev Guide](../#readme)
* [ ] Have you lint your code locally prior to submission (`npm run lint`)?
* [ ] Have you run prettier (`npm run format`)?
* [ ] Have you checked that app can be built (`npm run build`)?
* [ ] Have you checked that the build output looks according to design (screen, mobile, desktop) in Figma (`npm run serve`)?

### CODE REVIEW

File: src/components/Layout/Navigation/NavMenu/CommunityMenu/CommunityMenu.tsx [hunk 1]
```diff
@@ -3,20 +3,35 @@ import { useAtom } from 'jotai';
...
+
+  const formatCommunities = () => {
+    return communities.map(community => ({
+      title: community.title,
+      link: community.link,
+      iconClass: 'community',
+      iconProps: {
+        style: {
+          '--icon': `url('${community.icon.url}')`,
+          '--hoverIcon': `url('${community.hoverIcon.url}')`,
```
Comment (ViktorSoroka07) [inline src/components/Layout/Navigation/NavMenu/CommunityMenu/CommunityMenu.tsx:30]: Interesting approach

**Review by ViktorSoroka07 — DISMISSED** [review]

File: src/components/Layout/Navigation/NavMenu/CommunityMenu/CommunityMenu.tsx (hunk 1 above)
Comment (yelyzavetakhokhlova) [reply src/components/Layout/Navigation/NavMenu/CommunityMenu/CommunityMenu.tsx:30]: I wanted to use data attributes to set background like
`background-image: attr(data-icon);`
but turns out CSS doesn't support that and the only other way apart from this one was to use js

Comment (ViktorSoroka07) [reply src/components/Layout/Navigation/NavMenu/CommunityMenu/CommunityMenu.tsx:30]: I like it. Can you please adjust here to use your approach?

https://github.com/reportportal/reportportal.github.io/blob/develop/src/containers/CasesPage/CasesPage.tsx#L51

**Review by AmsterGet — DISMISSED** [review]

**Comment by AmsterGet** [comment]
I just noticed one thing

Could you please apply [these](https://github.com/reportportal/reportportal.github.io/pull/447/files#diff-0741bd992e82948a79c5ed1a018359a6f0430a7af46cb48158c7e43ef4a5cefdR8-R16) changes to the Contentful content?

**Review by ViktorSoroka07 — APPROVED** [review]

**Pull request merged**